pkg/network/setup/netstat_test.go

```diff
@@ -463,6 +463,53 @@ var _ = Describe("netstat", func() {
 		})
 	})
 
+	Describe("primary interface position in spec", func() {
+		const (
+			prNetworkName   = "primary"
+			prMAC           = "1C:CE:C0:01:BE:E7"
+			secNetworkName1 = "secondary1"
+			secMAC1         = "1C:CE:C0:01:BE:E9"
+			secNetworkName2 = "secondary2"
+			secMAC2         = "1C:CE:C0:01:BE:E8"
+		)
+
+		BeforeEach(func() {
+			setup.Vmi.Spec.Networks = append(setup.Vmi.Spec.Networks, newVMISpecPodNetwork(prNetworkName), newVMISpecMultusNetwork(secNetworkName1))
```

Comment:
`testSetup` has helper functions that takes care of the details on how an interface/network is added.
Could you please use those instead?

---

I wanted to suggest to improve a bit the tests readability below by:
- In the setup fixture, to add two networks/interfaces (always) that are non-primary.
- In the tests, to inset the primary network, once in the head, then at the tail and lastly in the middle.
- The expected list can be defined at the setup fixture, once.

Reply:
Fixed. I use indexes Entry instead of manipulation arrays since arrays are defined in BeforeEach and will not be initialized before running stage